parse.py

```diff
@@ -152,91 +151,83 @@ def printOutput(parser, document, opts):
 
 
 def getOptParser():
```

Comment:
Is it worth renaming this to (for example) `getArgParser`, and have `getOptParser` call the new one with a deprecation warning?

Reply:
I agree. I've named it `get_parser()` as I don't think the name needs to embed the implementation choice.

I didn't add a compatibility shim as I don't believe this is a public interface. But if I'm wrong and the project maintainers would like to see a shim here, please let me know and I'll ad it.